crates/ruff_linter/src/checkers/ast/analyze/expression.rs

```diff
@@ -869,6 +869,9 @@ pub(crate) fn expression(expr: &Expr, checker: &mut Checker) {
             if checker.enabled(Rule::DictGetWithNoneDefault) {
                 flake8_simplify::rules::dict_get_with_none_default(checker, expr);
             }
+            if checker.enabled(Rule::ZipDictKeysAndValues) {
+                flake8_simplify::rules::zip_dict_keys_and_values(checker, expr);
```

Comment:
Nit: for new rules, we're preferring to pass in `call`, rather than `expr`, since it ensures that we don't pass in expressions of the wrong type.